kratos/solving_strategies/schemes/residual_based_implicit_time_scheme.h

```diff
@@ -203,177 +212,172 @@ class ResidualBasedImplicitTimeScheme
      * @param rCurrentProcessInfo The current process info instance
      */
-
     void Calculate_RHS_Contribution(
-        Element::Pointer pElement,
-        LocalSystemVectorType& RHS_Contribution,
-        Element::EquationIdVectorType& EquationId,
+        Element::Pointer pCurrentElement,
+        LocalSystemVectorType& rRHS_Contribution,
+        Element::EquationIdVectorType& rEquationId,
         ProcessInfo& rCurrentProcessInfo
         ) override
     {
         KRATOS_TRY;
 
-        const std::size_t this_thread = OpenMPUtils::ThisThread();
+        const IndexType this_thread = OpenMPUtils::ThisThread();
 
         // Initializing the non linear iteration for the current element
-        // pElement->InitializeNonLinearIteration(rCurrentProcessInfo);
+        // pCurrentElement->InitializeNonLinearIteration(rCurrentProcessInfo);
 
         // Basic operations for the element considered
-        pElement->CalculateRightHandSide(RHS_Contribution,rCurrentProcessInfo);
+        pCurrentElement->CalculateRightHandSide(rRHS_Contribution,rCurrentProcessInfo);
 
-        pElement->CalculateMassMatrix(mMatrix.M[this_thread], rCurrentProcessInfo);
+        pCurrentElement->CalculateMassMatrix(mMatrix.M[this_thread], rCurrentProcessInfo);
 
-        pElement->CalculateDampingMatrix(mMatrix.D[this_thread],rCurrentProcessInfo);
+        pCurrentElement->CalculateDampingMatrix(mMatrix.D[this_thread],rCurrentProcessInfo);
 
-        pElement->EquationIdVector(EquationId,rCurrentProcessInfo);
+        pCurrentElement->EquationIdVector(rEquationId,rCurrentProcessInfo);
 
-        AddDynamicsToRHS (pElement, RHS_Contribution, mMatrix.D[this_thread], mMatrix.M[this_thread], rCurrentProcessInfo);
+        AddDynamicsToRHS (pCurrentElement, rRHS_Contribution, mMatrix.D[this_thread], mMatrix.M[this_thread], rCurrentProcessInfo);
 
-        KRATOS_CATCH( "" );
+        KRATOS_CATCH("ResidualBasedImplicitTimeScheme.Calculate_RHS_Contribution");
     }
 
     /**
      * @brief Functions totally analogous to the precedent but applied to the "condition" objects
-     * @param pCondition The condition to compute
-     * @param LHS_Contribution The LHS matrix contribution
-     * @param RHS_Contribution The RHS vector contribution
-     * @param EquationId The ID's of the element degrees of freedom
+     * @param pCurrentCondition The condition to compute
+     * @param rLHS_Contribution The LHS matrix contribution
+     * @param rRHS_Contribution The RHS vector contribution
+     * @param rEquationId The ID's of the element degrees of freedom
      * @param rCurrentProcessInfo The current process info instance
      */
-
     void Condition_CalculateSystemContributions(
-        Condition::Pointer pCondition,
-        LocalSystemMatrixType& LHS_Contribution,
-        LocalSystemVectorType& RHS_Contribution,
-        Element::EquationIdVectorType& EquationId,
+        Condition::Pointer pCurrentCondition,
```

Comment:
for the future (not in this PR) we should remove the Condition_    prefix from the name